code/modules/mob/living/simple_animal/hostile/true_changeling.dm

```diff
@@ -0,0 +1,195 @@
+#define TRUE_CHANGELING_REFORM_THRESHOLD 0 //Can turn back at will, by default
+#define TRUE_CHANGELING_PASSIVE_HEAL 3 //Amount of brute damage restored per tick
+#define TRUE_CHANGELING_FORCED_REFORM 180 //3 minutes
+
+//Changelings in their true form.
+//Massive health and damage, but move slowly.
+
+/mob/living/simple_animal/hostile/true_changeling
+	name = "true changeling"
+	real_name = "true changeling"
+	desc = "Holy shit, what the fuck is that thing?!"
+	speak_emote = list("says with one of its faces")
+	emote_hear = list("says with one of its faces")
+	icon = 'icons/mob/animal.dmi'
+	icon_state = "horror1"
+	icon_living = "horror1"
+	icon_dead = "horror_dead"
+	speed = -0.5
+	gender = NEUTER
+	a_intent = "harm"
+	stop_automated_movement = 1
+	status_flags = CANPUSH
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	minbodytemp = 0
+	health = 250
+	maxHealth = 250 //pretty durable
+	force_threshold = 10
+	healable = 0
+	environment_smash = 1 //Tables, closets, etc.
+	melee_damage_lower = 35
+	melee_damage_upper = 35
+	see_in_dark = 8
+	see_invisible = SEE_INVISIBLE_MINIMUM
+	wander = 0
+	attacktext = "tears into"
+	attack_sound = 'sound/creatures/hit3.ogg'
+	butcher_results = list(/obj/item/reagent_containers/food/snacks/meat/slab = 15) //It's a pretty big dude. Actually killing one is a feat.
+	gold_core_spawnable = 1 //To be fair, they won't be able to become changelings, like headslugs can
+	var/time_spent_as_true = 0
+	var/playstyle_string = "<b><font size=3 color='red'>We have entered our true form!</font> We are unbelievably powerful, and regenerate life at a steady rate. However, most of \
+	our abilities are useless in this form, and we must utilise the abilities that we have gained as a result of our transformation. Taking too much damage will also turn us back into a \
+	human in addition to knocking us out. Finally, we will uncontrollably revert into a human after some time due to our inability to maintain this form.</b>"
+	var/mob/living/carbon/human/stored_changeling = null //The changeling that transformed
+	var/devouring = FALSE //If the true changeling is currently devouring a human
+	var/wallcrawl = FALSE //If the true changeling is crawling around the place, allowing it to counteract gravity loss
+
+/mob/living/simple_animal/hostile/true_changeling/New()
+	..()
+	var/horrorskin = pick(1, 2, 3, 4)
+	switch(horrorskin)
+		if(1)
+			icon_state = "horror1"
+		if(2)
+			icon_state = "horror2"
+		if(3)
+			icon_state = "horror3"
+		if(4)
+			icon_state = "horror4"
+	playsound(usr, 'sound/creatures/ling_scream.ogg', 50, 1)
+	spawn(0)
+		to_chat(usr, playstyle_string)
```

Comment:
Move this to Login. I thought i already posted this comment but it seems to be missing ?